pkg/sql/alter_type.go

```diff
@@ -492,7 +492,7 @@ func (p *planner) canModifyType(ctx context.Context, desc *typedesc.Mutable) err
 	}
 	if !hasOwnership {
 		return pgerror.Newf(pgcode.InsufficientPrivilege,
-			"must be owner of type %s", tree.Name(desc.GetName()))
+			"must be owner of type %q", tree.Name(desc.GetName()))
```

Comment:
I'd sort of prefer we kept these and change the declarative schema changer to use `%s and the `tree.Name` type which will quote identifiers it needs to quote in its string method.